images/macos/provision/utils/utils.sh

```diff
@@ -63,32 +63,16 @@ is_Catalina() {
     fi
 }
 
-is_Mojave() {
-    if [ "$OSTYPE" = "darwin18" ]; then
-        true
-    else
-        false
-    fi
-}
-
-is_Less_Catalina() {
-    if is_Mojave; then
-        true
-    else
-        false
-    fi
-}
-
 is_Less_BigSur() {
```

Comment:
Do we need this now? It is the same as IsCatalina

Reply:
That's a good idea, thank you.